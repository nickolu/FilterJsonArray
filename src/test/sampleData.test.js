const sampleData = require('./sampleData.js');

describe('sampleData',()=>{
  it('should not have changed',()=>{
    expect(sampleData).toMatchSnapshot();
  });
});
