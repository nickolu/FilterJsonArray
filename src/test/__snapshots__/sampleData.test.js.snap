// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`sampleData should not have changed 1`] = `
Object {
  "simpleDataArrays": Object {
    "long": Array [
      Object {
        "a": "one",
        "b": "four",
        "c": "seven",
      },
      Object {
        "a": "two",
        "b": "five",
        "c": "eight",
      },
      Object {
        "a": "three",
        "b": "six",
        "c": "nine",
      },
      Object {
        "a": "ten",
        "b": "eleven",
        "c": "twelve",
      },
      Object {
        "a": "thirteen",
        "b": "fourteen",
        "c": "fifteen",
      },
      Object {
        "a": "sixteen",
        "b": "seventeen",
        "c": "eightteen",
      },
    ],
    "numbers": Array [
      Object {
        "a": "1",
        "b": "7",
        "c": "13",
      },
      Object {
        "a": "2",
        "b": "8",
        "c": "14",
      },
      Object {
        "a": "3",
        "b": "9",
        "c": "15",
      },
      Object {
        "a": "4",
        "b": "10",
        "c": "16",
      },
      Object {
        "a": "5",
        "b": "11",
        "c": "17",
      },
      Object {
        "a": "6",
        "b": "12",
        "c": "18",
      },
    ],
    "short": Array [
      Object {
        "a": "one",
        "b": "four",
        "c": "seven",
      },
      Object {
        "a": "two",
        "b": "five",
        "c": "eight",
      },
      Object {
        "a": "three",
        "b": "six",
        "c": "nine",
      },
    ],
  },
  "spellsArray": Array [
    Object {
      "casting_time": "1 action",
      "class": Object {
        "Sorcerer": "yes",
        "Wizard": "yes",
      },
      "components": "VSM",
      "concentration": "no",
      "description": "You draw the moisture from every creature in a 30-foot cube centered on a point you choose within range. Each creature in that area must make a Constitution saving throw. Constructs and undead aren't affected, and plants and water elementals make this saving throw with disadvantage. A creature takes 10d8 necrotic damage on a failed save, or half as much damage on a successful one. ",
      "duration": "Instantaneous",
      "level": "8th",
      "name": "Abi-Dalzim's Horrid Wilting",
      "page": "ee 15",
      "range": "150 feet",
      "ritual": "no",
      "school": "Necromancy",
    },
    Object {
      "casting_time": "1 reaction",
      "class": Object {
        "Druid": "yes",
        "Ranger": "yes",
        "Wizard": "yes",
      },
      "components": "S",
      "concentration": "no",
      "description": "The spell captures some of the incoming energy, lessening its effect on you and storing it for your next melee attack. You have resistance to the triggering damage type until the start of your next turn. Also, the first time you hit with a melee attack on your next turn, the target takes an extra 1d6 damage of the triggering type, and the spell ends. <br><br><strong>At Higher Levels.</strong> When you cast this spell using a spell slot of 2nd level or higher, the extra damage increases by 1d6 for each slot level above 1st. ",
      "duration": "1 round",
      "level": "1st",
      "name": "Absorb Elements",
      "page": "ee 15",
      "range": "Self",
      "ritual": "no",
      "school": "Abjuration",
    },
    Object {
      "casting_time": "1 action",
      "class": Object {
        "Sorcerer": "yes",
        "Wizard": "yes",
      },
      "components": "VS",
      "concentration": "no",
      "description": "You hurl a bubble of acid. Choose one creature within range, or choose two creatures within range that are within 5 feet of each other. A target must succeed on a Dexterity saving throw or take 1d6 acid damage. <br />This spell's damage increases by 1d6 when you reach 5th level (2d6), 11th level (3d6), and 17th level (4d6). ",
      "duration": "Instantaneous",
      "level": "Cantrip",
      "name": "Acid Splash",
      "page": "phb 211",
      "range": "60 feet",
      "ritual": "no",
      "school": "Conjuration",
    },
    Object {
      "casting_time": "1 action",
      "class": Object {
        "Sorcerer": "yes",
        "Wizard": "yes",
      },
      "components": "VSM",
      "concentration": "no",
      "description": "A line of roaring flame 30 feet long and 5 feet wide emanates from you in a direction you choose. Each creature in the line must make a Dexterity saving throw. A creature takes 3d8 fire damage on a failed save, or half as much damage on a successful one. <br><br><strong>At Higher Levels.</strong> When you cast this spell using a spell slot of 3rd level or higher, the damage increases by 1d8 for each slot level above 2nd. ",
      "duration": "Instantaneous",
      "level": "2nd",
      "name": "Aganazzar's Scorcher",
      "page": "ee 15",
      "range": "30 feet",
      "ritual": "no",
      "school": "Evocation",
    },
    Object {
      "casting_time": "1 action",
      "class": Object {
        "Cleric": "yes",
        "Paladin": "yes",
      },
      "components": "VSM",
      "concentration": "no",
      "description": "Your spell bolsters your allies with toughness and resolve. Choose up to three creatures within range. Each target's hit point maximum and current hit points increase by 5 for the duration. <br><br><strong>At Higher Levels.</strong> When you cast this spell using a spell slot of 3 rd level or higher, a target's hit points increase by an additional 5 for each slot level above 2nd. ",
      "duration": "8 hours",
      "level": "2nd",
      "name": "Aid",
      "page": "phb 211",
      "range": "30 feet",
      "ritual": "no",
      "school": "Abjuration",
    },
    Object {
      "casting_time": "1 minute",
      "class": Object {
        "Ranger": "yes",
        "Wizard": "yes",
      },
      "components": "VSM",
      "concentration": "no",
      "description": "You set an alarm against unwanted intrusion. Choose a door, a window, or an area within range that is no larger than a 20-foot cube. Until the spell ends, an alarm alerts you whenever a Tiny or larger creature touches or enters the warded area. When you cast the spell, you can designate creatures that won't set off the alarm. You also choose whether the alarm is mental or audible. <br / >A mental alarm alerts you with a ping in your mind if you are within 1 mile of the warded area. This ping awakens you if you are sleeping. An audible alarm produces the sound of a hand bell for 10 seconds within 60 feet. ",
      "duration": "8 hours",
      "level": "1st",
      "name": "Alarm",
      "page": "phb 211",
      "range": "30 feet",
      "ritual": "yes",
      "school": "Abjuration",
    },
  ],
}
`;
